### PR TITLE
2020 fall dev01

### PR DESCRIPTION
**Pull request by midran**
Development for fall release 2020. Most issues are in but not all.

### CODE REVIEW

**Review by jerouris — CHANGES_REQUESTED** [review]
Please add a test for checking that 9958 is no longer valid

File: guide/release-notes/v3.0.7.adoc [hunk 1]
```diff
@@ -12,6 +12,6 @@ Release date:: 2020-11-01
 * Rule PEPPOL-COMMON-R040: "GLN must have a valid format according to GS1 rules". Changed from severity "warning" to "fatal". (The rule was introduced in fall 2019 with severity "warning" to avoid disruptions but with the intention to be changed to severity "fatal" after 6-12 months). Issue: [POACC-333] and [POACC-341]
 * EAS code list, added code 0209, removed code 9958. Rule updated accordingly.
 * ICD code list, added code 0209. Rule updated accordingly. Names and descriptions added where missing.
-* VATEX code list, added code VATEX-EU-79C for repayment of expenditure. Rule updated accordingly.
+* VATEX code list, added code VATEX-EU-79C for repayment of expenditure. Codes put in upper case.
```
Comment (jerouris) [inline guide/release-notes/v3.0.7.adoc:15]: Was the VATEX code list previously used with low letter codes? This could create a backwards incompatibility

Comment (ConnectingEurope) [reply guide/release-notes/v3.0.7.adoc:15]: Yes the VATEX code was previously in low letter but this was changed to align with EU Publication Office conventions. The validation rule in TS434 is however implemented as case neutral to provide backwards compatiblility. This change from lower to upper case. This was noted in the Peppol spring release by referencing the TC434 / CEF update to the EN16931 rules, https://github.com/ConnectingEurope/eInvoicing-EN16931/releases/tag/validation-1.3.1

**Review by jerouris — DISMISSED** [review]

File: guide/release-notes/v3.0.8.adoc [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+* Migration plan from BIS2 to BIS3 removed. Is completed.
+* Use GLN as the only examlpe in section 10.2. Delivery Details (Date and Location). Removed reference to GSRN from the description text. [POACC-332]
+
+== Changes to support documents
+
+
+== Changes to code lists and validation artefacts
+* Rule PEPPOL-COMMON-R040: "GLN must have a valid format according to GS1 rules". Changed from severity "warning" to "fatal". (The rule was introduced in fall 2019 with severity "warning" to avoid disruptions but with the intention to be changed to severity "fatal" after 6-12 months). Issue: [POACC-333] and [POACC-341]
+* EAS code list, added code 0209, removed code 9958. Rule updated accordingly.
+* ICD code list, added code 0209. Rule updated accordingly. Names and descriptions added where missing.
```
Comment (jerouris) [inline guide/release-notes/v3.0.8.adoc:16]: Code lists in https://github.com/OpenPEPPOL/peppol-bis-invoice-3/tree/2020-Fall-Dev01/rules/sch do not appear to have this rule applied. (Both Peppol and EN)

File: guide/release-notes/v3.0.8.adoc [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+* Definitions for EndPoint identifiers BT-34 and BT-49 aligned with defintion in EN16931. Issue: [POACC-331]
+* Migration plan from BIS2 to BIS3 removed. Is completed.
+* Use GLN as the only examlpe in section 10.2. Delivery Details (Date and Location). Removed reference to GSRN from the description text. [POACC-332]
+
+== Changes to support documents
+
+
+== Changes to code lists and validation artefacts
+* Rule PEPPOL-COMMON-R040: "GLN must have a valid format according to GS1 rules". Changed from severity "warning" to "fatal". (The rule was introduced in fall 2019 with severity "warning" to avoid disruptions but with the intention to be changed to severity "fatal" after 6-12 months). Issue: [POACC-333] and [POACC-341]
+* EAS code list, added code 0209, removed code 9958. Rule updated accordingly.
```
Comment (jerouris) [inline guide/release-notes/v3.0.8.adoc:15]: Code lists in https://github.com/OpenPEPPOL/peppol-bis-invoice-3/tree/2020-Fall-Dev01/rules/sch do not appear to have this rule applied. (Both Peppol and EN)

**Review by jerouris — CHANGES_REQUESTED** [review]
CEN-EN16931-CII.sch is an external file and should not be changed.

**Pull request merged**